federation/pkg/kubefed/init/init.go

```diff
@@ -125,6 +125,10 @@ var (
 		"app":    "federated-cluster",
 		"module": "federation-controller-manager",
 	}
+
+	validDNSProviders = []string{
```

Comment:
Just looked at it. But seems like it's broken on the tests. It only returns [coredns]